src/main/java/com/enonic/app/rewrite/PathBuilder.java

```diff
@@ -0,0 +1,36 @@
+package com.enonic.app.rewrite;
+
+public class PathBuilder
+{
+    private final String context;
+
+    private final String path;
+
+    private PathBuilder( final String context, final String path )
+    {
+        this.context = normalize( context );
+        this.path = normalize( path );
+    }
+
+    public static PathBuilder create( final String context, final String path )
+    {
+        return new PathBuilder( context, path );
+    }
+
+    private String normalize( String value )
+    {
+        if ( value != null )
+        {
+            return value.startsWith( "/" ) ? value.replaceFirst( "/", "" ) : value;
+        }
+        else
+        {
+            return "";
```

Comment:
Why would ever be null here? It should be handled at earlier stager, probably and not normalized to empty string ever.

Reply:
done